init/retina/main_linux.go

```diff
@@ -21,23 +24,31 @@ var (
 func main() {
 	// Initialize logger
 	opts := log.GetDefaultLogOpts()
+	zl, err := log.SetupZapLogger(opts)
+	if err != nil {
+		panic(err)
+	}
+
+	// Define a command-line flag "config" with a default value and parse the flags, then create a logger with a name and version.
+	configPath := flag.String("config", "/retina/config/config.yaml", "path to the config file")
```

Comment:
@Carpe-Wang looks like you need to pass in the path to the config via arguments to the init-container as well. Please refer to how we are currently doing it for the agent's main container
![image](https://github.com/microsoft/retina/assets/28567936/63bae1b6-ae9b-45dd-8b5f-dad6d2edf296)

Reply:
<img width="714" alt="image" src="https://github.com/microsoft/retina/assets/78642589/f377b79e-46e9-4a0a-af65-6b0cd393fcd1">


---------------
I added the following section to the relevant file:
```
args:
  - --config=/retina/config/config.yaml
```
Does this meet your expectation?"

Reply:
that should work

Reply:
okay, I had push the newest commit.

Reply:
oops i forgot to mention, you will also have to mount the path the config file like we are doing with the main container. Thanks for the prompt response!

Reply:
@nddq I change the file according to your comment. Please review it.